src/pyodide.js

```diff
@@ -32,6 +32,49 @@ var languagePluginLoader = new Promise((resolve, reject) => {
     }
   };
 
+  // clang-format off
+  let preloadWasm = () => {
+    // On Chrome, we have to instantiate wasm asynchronously. Since that
+    // can't be done synchronously within the call to dlopen, we instantiate
+    // every .so that comes our way up front, caching it in the
+    // `preloadedWasm` dictionary.
```

Comment:
Given that we only mount the directories with the packages that we use, this won't have much performance overhead would it? Unless of course the package is very large with multiple unrelated parts (e.g. scipy) ?

Reply:
The current version of Pyodide is already doing something like this on Chrome (it's just that's it's happening earlier on as part of the "preload" system built into emscripten).  This is just a slightly different implementation to happen later after the ability to decompress files is available.  There is some additional overhead because now we are decompressing the wasm content, but that should be minimal.

The place where we run into issues with this approach now is with Pandas, since it has so many large Cython-based `.so` files, only some of which ever get used.